lib/features/context-pad/ContextPad.js

```diff
@@ -442,6 +446,12 @@ ContextPad.prototype.close = function() {
     return;
   }
 
+  if (this._timeout) {
+    clearTimeout(this._timeout);
+
+    this._timeout = null;
```

Comment:
What is the reason we set the timeout to null? As I can test, `clearTimeout` works fine with an already canceled timeout, so we could reduce complexity in our logic and just cancel all the time

Reply:
I'm also fine with merging it as-is if it is intended